src/en/charms-storage.md

```diff
@@ -304,6 +305,15 @@ config:
   security.privileged: "true"
 ```
 
+### Remove storage
+
+Specify one or more unit storage, volume, or filesystem IDs, which can be found
+in the output of `juju storage`, like this:
+
+```bash
+juju remove-storage pgdata/0
```

Comment:
It would be good to provide more examples (e.g. one of each type if possible). E.g. demonstrate the use of 'show storage' with some output, then examples below of removing the storage based on different types of id.